src/commands/build.rs

```diff
@@ -14,5 +14,25 @@ pub fn run(matches: &ArgMatches) -> Result<(), failure::Error> {
     let manifest = Manifest::new(&config_path)?;
     let env = matches.value_of("env");
     let target = &manifest.get_target(env, false)?;
-    build(&target)
+    if matches.is_present("output") {
+        if matches.value_of("output") == Some("json") {
+            terminal::message::set_output_type(terminal::message::OutputType::Json)
+        } else {
+            terminal::message::user_error("json is the only valid value for output flag");
```

Comment:
code maybe works as is? i'd double check this is off the top of my head..  - we want the flag to be highlighted yellow

```suggestion
            let flag = terminal::message::highlight("`--output`");
            terminal::message::user_error(&format!("{} must be json", flag));
```

Reply:
I found the `clap` option for possible values to do this.